example/gluon/word_language_model/train.py

```diff
@@ -188,11 +189,7 @@ def train():
             print('test loss %.2f, test ppl %.2f'%(test_L, math.exp(test_L)))
         else:
             args.lr = args.lr*0.25
-            trainer._init_optimizer('sgd',
-                                    {'learning_rate': args.lr,
-                                     'momentum': 0,
-                                     'wd': 0})
-            model.collect_params().load(args.save, context)
+            trainer.set_learning_rate(args.lr)
```

Comment:
The original logic will decay learning rate, and reload previous saved parameters to continue the training. Why the reloading is removed?

Reply:
There is evidence that reloading previous saved parameter is better. Keeping the current parameter is instead theoretically supported in the view of optimization.